packaging/RPMS/Fedora/rabbitmq-server.spec

```diff
@@ -234,6 +234,9 @@ systemctl try-restart %{name}.service >/dev/null 2>&1 || :
 rm -rf %{buildroot}
 
 %changelog
+* Wed Dec 13 2017 info@rabbitmq.com 3.7.1~beta.1-1
```

Comment:
Is the change log update intentional?

Reply:
It got pulled in when I merged my branch that originated at `v3.7.x` into this one.

Reply:
Fair enough.